ragna/deploy/_ui/components/metadata_filters_builder.py

```diff
@@ -0,0 +1,307 @@
+from datetime import datetime
+
+import panel as pn
+import param
+
+from ragna.core._metadata_filter import MetadataFilter
+
+METADATA_FILTERS = {
+    "document_name": {"type": str},
+    "size": {"type": int},
+    "extension": {"type": str},
```

Comment:
We can keep this for now as it is supplied by our default `LocalDocument`, but they have to be variable before we can merge into `main`.